.gitignore

```diff
@@ -124,3 +124,4 @@ packages/datadog-plugin-next/test/yarn.lock
 packages/dd-trace/test/appsec/next/*/package.json
 packages/dd-trace/test/appsec/next/*/node_modules
 packages/dd-trace/test/appsec/next/*/yarn.lock
+!packages/dd-trace/**/telemetry/logs
```

Comment:
Why is this needed?

Reply:
because `logs` is globally ignored a few lines above